app/views/permissions/_hidden_collections_notification.html.erb

```diff
@@ -0,0 +1,7 @@
+<% unless @hidden_entry_titles.blank? %>
```

Comment:
We'll need to run this through @mreese84 and/or @jsiarto -- I don't think we want this type of messaging.